src/OpenTelemetry.AutoInstrumentation/Configuration/PluginsConfigurationHelper.cs

```diff
@@ -63,7 +66,8 @@ private static TracerProviderBuilder InvokePlugin(this TracerProviderBuilder bui
         var mi = t.GetMethod(configureTracerProviderMethodName, new Type[] { typeof(TracerProviderBuilder) });
         if (mi is null)
         {
-            throw new MissingMethodException(t.Name, configureTracerProviderMethodName);
+            Logger.Warning(new MissingMethodException(t.Name, configureTracerProviderMethodName), $"{configureTracerProviderMethodName} is missing in {pluginAssemblyQualifiedName}");
```

Comment:
This might not be possible. As we really the key name `OTEL_DOTNET_AUTO_PLUGINS` is same for all signals. We need to attempt to plugin load on all 3 signals, if we don't find a specific method we need to move on without throwing an error message.

Reply:
IMO it should be informational level, without logging exception.
It is completely valid scenario by current desing.

Reply:
my typo made things sound different than what I intended, it should have been something like

> if we have a single **method** invoking **all** plugins for all 3 kinds of signals.

So basically a single place to handle all the plugins instead of having per signal as right now. Anyway, it is a refactor that goes beyond the scope of this PR.